ui/src/core/omnibox_manager.ts

```diff
@@ -111,6 +111,19 @@ export class OmniboxManagerImpl implements OmniboxManager {
     return this._statusMessageContainer.msg;
   }
 
+  // Returns true if the topbar should be visible in zen mode.
```

Comment:
I'm not sure why this getter should live in the omnibox manager. Seems like this logic should just go in the omnibox mithril component as all this state is publicly available.